db/seeds.rb

```diff
@@ -19,60 +19,131 @@
+trip7 = Trip.new(destination_id: 1,activity_id: 2,name: "The Colorado Rapids",
+                 description: "Check an item off your bucket list with this once in a lifetime trip down the Colorado River.",
+                 trip_cost: 9550.00)
+trip7.image = image7
+trip7.save
+
+image8 = File.open('app/assets/images/basez.jpg')
+trip8 = Trip.new(destination_id: 6,activity_id: 5,name: "Base Jumping the Fjords",
+                 description: "Performing the most daring of stunts in the most isolated of places.",
+                 trip_cost: 38900.00)
+trip8.image = image8
+trip8.save
+
+image9 = File.open('app/assets/images/dolomites.jpg')
+trip9 = Trip.new(destination_id: 3,activity_id: 1,name: "Huts in the Dolomites",
+                 description: "Experience the old world culture of Northern Italy.",
+                 trip_cost: 9400.00)
+trip9.image = image9
+trip9.save
+
+image10 = File.open('app/assets/images/yosemite.jpg')
+trip10 = Trip.new(destination_id: 1,activity_id: 4,name: "Yosemite Granite",
+                 description: "Experience the world's natural playground and climb the classic Yosemite granite domes.",
+                 trip_cost: 21500.00)
+trip10.image = image10
+trip10.save
+
+image11 = File.open('app/assets/images/atlas.jpg')
+trip11 = Trip.new(destination_id: 5,activity_id: 4,name: "Climbing the High Atlas",
+                  description: "Spend your time in the thin air of the High Atlas Mountains in Northern Morocco.",
+                  trip_cost: 17900.00)
+trip11.image = image11
+trip11.save
+
+image12 = File.open('app/assets/images/montana.jpg')
+trip12 = Trip.new(destination_id: 1,activity_id: 6,name: "Hunting in Montana",
+                  description: "Experience one of the most vibrant displays of wildlife in the continental United States.",
+                  trip_cost: 33050.00)
+trip12.image = image12
+trip12.save
+
+image13 = File.open('app/assets/images/navua.jpg')
+trip13 = Trip.new(destination_id: 6,activity_id: 2,name: "The Navua River",
+                  description: "What's that? You didn't know that the tiny island of Fiji had such awesome river rafting? Well it does! Ride these rapids 65 kilometers to the south coast.",
+                  trip_cost: 25850.00)
+trip13.image = image13
+trip13.save
+
+image14 = File.open('app/assets/images/positano.jpg')
+trip14 = Trip.new(destination_id: 3,activity_id: 3,name: "Riviera Diving",
+                  description: "See the fishes below the water, eat the fishes above the water in this Italian getaway.",
+                  trip_cost: 21000.00)
+trip14.image = image14
+trip14.save
+
+image15 = File.open('app/assets/images/buffalo.jpg')
+trip15 = Trip.new(destination_id: 6,activity_id: 6,name: "Outback Big Game",
+                  description: "Go down under and take on some big game in the rugged Australian Outback.",
+                  trip_cost: 49500.00)
+trip15.image = image15
+trip15.save
+
+image16 = File.open('app/assets/images/zambezi.jpg')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.